Libplanet/Consensus/VoteSet.cs

```diff
@@ -73,19 +93,34 @@ public bool Add(Vote vote)
             }
         }
 
+        /// <summary>
+        /// Check if the validator who voted has more than +2/3.
+        /// </summary>
+        /// <returns><c>true</c> if +2/3 of the users in the <see cref="ValidatorSet"/> voted.
+        /// <c>false</c> otherwise.</returns>
         public bool HasTwoThirdAny()
         {
             var twoThird = ValidatorSet.Length * 2.0 / 3.0;
             return _votes.Count(x => !(x.Value.Signature is null)) > twoThird;
         }
 
+        /// <summary>
+        /// Check if the validator who voted about Prevote state has more than +2/3.
+        /// </summary>
+        /// <returns><c>true</c> if +2/3 of the users in the <see cref="ValidatorSet"/> voted.
+        /// about <see cref="VoteFlag.Absent"/> <c>false</c> otherwise.</returns>
         public bool HasTwoThirdPrevote()
         {
             var twoThird = ValidatorSet.Length * 2.0 / 3.0;
             return _votes.Count(x =>
                 !(x.Value.Signature is null) && x.Value.Flag == VoteFlag.Absent) > twoThird;
         }
 
+        /// <summary>
+        /// Check if the validator who voted about Precommit state has more than +2/3.
+        /// </summary>
+        /// <returns><c>true</c> if +2/3 of the users in the <see cref="ValidatorSet"/> voted.
+        /// about <see cref="VoteFlag.Commit"/> <c>false</c> otherwise.</returns>
```

Comment:
```suggestion
        /// <returns><c>true</c> if +2/3 of the users in the <see cref="ValidatorSet"/> voted
        /// about <see cref="VoteFlag.Commit"/>. <c>false</c> otherwise.</returns>
```